lib/lintbot.js

```diff
@@ -43,8 +42,8 @@ async function lintbot({
     process.argv = process.argv.filter(a => !a.startsWith('-'));
 
     if (process.argv.length < 1) {
-      Logger.error(usage);
-      return 1;
+  Logger.error(usage);
```

Comment:
:stop_sign: **Lint Error** Expected indentation of 6 spaces but found 2.

```suggestion
      Logger.error(usage);
```

[indent](https://www.google.com/search?btnI&q=eslint%20rule%20indent)